sources/lib/jam/jam-state.dylan

```diff
@@ -12,6 +12,7 @@ define class <jam-state> (<object>)
   constant slot %jam-targets :: <string-table> = make(<string-table>);
   constant slot %jam-random :: <random> = make(<random>);
   constant slot %jam-temporary-files :: <deque> = make(<deque>);
+  constant slot %jam-jamfiles :: <string-table> = make(<string-table>);
```

Comment:
It wouldn't hurt to have comments regarding types on some of these slot definitions.
```
// target name -> mod date
```